usecases/scheduler/src/lambda/ssv2_resultProcesser/resultProcesser.py

```diff
@@ -0,0 +1,263 @@
+import boto3 
+import json
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+import os
+import openpyxl
+import logging
+
+## Initialization
+SnsEmailSubject = "[AWS] Service Screener Scheduler Report"
+SnsEmailBodyPrefix = "Here is your Service Screener report.\n\n"
+snsPrefix = os.environ['SSV2_SNSARN_PREFIX']
+
+
+successResp = {
+    'statusCode': 200,
+    'body': 'success'
+}
+
+currentResults = {}
+previousResults = {}
+accounts = {}
+
+SHEETS_TO_SKIP = ['Info', 'Appendix']
+
+def lambda_handler(event, context):
+    record = event['detail']
+    region = event['region']
+    targetBucket = record['bucket']['name']
+    targetObject = record['object']['key']
+    
+    s3 = boto3.client('s3', region_name=region)
+    sns = boto3.client('sns', region_name=region)
+    
+    filepath = targetObject.split('/')
+    configId = filepath[0]
+    currentDate = filepath[1]
+
+    objs = s3.list_objects_v2(Bucket=targetBucket, Prefix=configId + '/' + currentDate + '/')
+    contents = objs.get('Contents')
+    for xlsx in contents:
+        if xlsx.get('Key').endswith('xlsx'):
+            _obj = xlsx.get('Key').split('/')
+            accounts[_obj[2]] = {'currentRun': currentDate}
+
+    # find latest available filesx
+    currentDateObj = datetime.strptime(currentDate, '%Y%m%d').date()
+    for x in range(3):
+        prefixSearch = (currentDateObj - relativedelta(months=x)).strftime("%Y%m")
+        objs = s3.list_objects_v2(Bucket=targetBucket, Prefix=configId + '/' + prefixSearch)
+        contents = objs.get('Contents')
+        if not contents:
+            continue 
+
+        contents.reverse()
+        for file in contents:
+            _file = file.get('Key')
+            if _file.endswith('xlsx'):
+                info = _file.split('/')
+                acct = info[2]
+                if acct in accounts and accounts[acct].get('previousRun') == None and info[1] != currentDate:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)

Similar issue at line numbers 64 and 73.